src/main.ts

```diff
@@ -2,6 +2,7 @@
 
 import './components/sidebar/sidebar.ts';
 import './components/header.ts';
+import './components/toast.ts';
```

Comment:
wäre nicht besser dynamic importiert in den Komponenten die tatsächlich einen toast verwenden?